cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -22,28 +22,34 @@ function _port() {
 function prepare_config() {
     BASE_PATH=${KUBEVIRTCI_CONFIG_PATH:-$PWD}
     cat >$BASE_PATH/$KUBEVIRT_PROVIDER/config-provider-$KUBEVIRT_PROVIDER.sh <<EOF
-master_ip=$(_main_ip)
-kubeconfig=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
-kubectl=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubectl
-gocli=${BASE_PATH}/../cluster-up/cli.sh
-docker_prefix=\${DOCKER_PREFIX:-localhost:$(_port registry)/kubevirt}
-manifest_docker_prefix=\${DOCKER_PREFIX:-registry:5000/kubevirt}
-EOF
+    master_ip=$(_main_ip)
```

Comment:
This indentation change  leads to the heredoc used not working. Therefore the k8s provisioning job fails. You should revert changes of this block.

Reply:
nice catch, thx

Reply:
there is some way to fix it with <<-
but didnt manage, so reverting and testing

the prow job will retest it right ?

Reply:
reverted